jenkins/manila-functional-tests/20-install-openstack.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -xue
+
+# Set working directory to where the fabfile resides.
+SCRIPT_DIR=$(dirname ${0})
+cd ${SCRIPT_DIR}
+
+# Remove any deployed resources on error.
+trap "set +u && source heat-venv/bin/activate && fab destroy" ERR
+
+source /tmp/manilaci-hosts
+git clone https://github.com/openstack-dev/devstack.git
+cp devstack/samples/local.conf devstack
```

Comment:
Start from a clean/empty local.conf. We don"t really know what's actually in devstack/samples/local.conf and it can change over time.